docs/_advanced/middleware_listener.md

```diff
@@ -8,15 +8,35 @@ order: 5
 <div class="section-content">
 Listener middleware is used for logic across many listener functions (but usually not all of them). They are added as arguments before the listener function in one of the built-in methods. You can add any number of listener middleware before the listener function.
 
-There’s a collection of built-in listener middleware that you can use like `subtype()` for filtering on a message subtype and `directMention()` which filters out any message that doesn’t directly @-mention your bot.
+There’s a collection of built-in listener middleware that you can use like `subtype()` for filtering on a message subtype and `directMention()` which filters out any message that doesn’t directly @-mention your bot "at the beginning of the message". 
 
-But of course, you can write your own middleware for more custom functionality. While writing your own middleware, your function must call `await next()` to pass control to the next middleware, or `throw` to pass an error back up the previously-executed middleware chain.
-
-As an example, let’s say your listener should only deal with messages from humans. You can write a listener middleware that excludes any bot messages.
+As an example, let's say your listener should only deal with messages that mention your bot directly.
```

Comment:
I'd add an extra sentence just like the other example that mentions direction mention.  Something like, You can use the directMention middleware when your app responds to messages.

Reply:
added explainer sentence